echidna/core/smear.py

```diff
@@ -109,16 +109,45 @@ class EnergySmearLY(Smear):
     """ The class which smears energy. It accepts resolution in terms of light
       yield (LY) in units of NHit per MeV.
 
+    Args:
+      poisson (bool): If True, use poisson smearing.
+
     Attributes:
       _light_yield (float): The light yield of the scintillator in NHits per
         MeV.
+      _poisson_smear (Bool): True if poisson smearing is to be applied. False
+      if gaussian smearing is to be applied.
```

Comment:
@EdLeming, just noticed that this doesn't get formatted correctly. With the sphinx docs, anything that is in a list like this should be indented by two spaces on line continuation. There were a couple of other places where the indentation is not correct.

At the moment it looks like this:
![screenshot from 2016-02-19 16 40 48](https://cloud.githubusercontent.com/assets/1931666/13182033/ba4e1812-d727-11e5-8de0-30b01f3c7499.png)